internal/contracts/contracts.go

```diff
@@ -274,4 +276,20 @@ func (mc *JSONContract) Unmarshal() (Contract, error) {
 		mc.StateNonce,
 	}
 	return c, nil
+	
+}
+
+
+
+func GenerateRandomContract() (*Contract,error){
+	b := make([]byte, 32)
+	rand.Read(b)
+	genSenderPrivKey, _ := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	genRecipPubKeyHash :=  b
+	genVersion := binary.LittleEndian.Uint16(b[:])
```

Comment:
Is there a way to make sure this is never 0? The only instance `New` returns an error is if the version passed in is a zero.

- [ ] Generate only nonzero numbers for version
- [ ] Make return type be just a `Contract`

Reply:
The new function returns 2 values, contract and error, so when I return a new *random contract, it also returns error

Reply:
I see. We can do what you had initially: define a return variable, with its error, then just return the return variable at the end.  It doesn't make much sense to generate a random contract and need to do error checks on the call.

Reply:
Also, the new function already checks if the version is greater than 0. do u want it so that is doesn't make a contract with a zero value version in the first place?

Reply:
Yes this is exactly what I want.